### ISSUE TITLE
Hangs on missing HTML files

### ISSUE THREAD

**Issue by TheHalcyonSavant** [vkadam/grunt-jsbeautifier#51]
When the specified files can't be found by the multitask `src` configuration pattern, then jsbeautifier task starts some infinite loop in the background and it doesn't return immediately. To reproduce this bug, just use the following simple HTML configuration:

```javascript
jsbeautifier: {
  html: {
    src: ["app/views/*.html"]
  }
}
```
and leave the `src` folder empty.
Now, try run `grunt jsbeautifier:html` and it'll never return - will hang forever.

**Comment by dpalic** [comment]
same issue here. We did a workaround by adding a dummy file into the empty folder, but this is a bad hack (even if it helps to workaround it)

**Comment by vkadam** [comment]
Fixed with upgrade of async dependency in commit e48b8fd264d44136b83bd4221d979beec54d2227, test added in commit ccfd245802b84c7291a00794de4fb30a5caecf6e. Will be available with version 0.2.12
